xml/Makefile.am

```diff
@@ -139,7 +140,7 @@ api/api-result-%.rng: $(API_files) Makefile.am
 	echo '      <attribute name="request"> <text /> </attribute>' >> $@
 	echo '      <optional>' >> $@
 	echo '        <choice>' >> $@
-	for rng in $(API_base); do $(top_srcdir)/xml/best-match.sh api/$$rng $(*) $(@) "          " || :; done
+	for rng in $(API_request_base); do $(top_srcdir)/xml/best-match.sh api/$$rng $(*) $(@) "          " || :; done
```

Comment:
we should eventually document that the same approaches apply
also to `api` ones, at least it seems so